app/assets/stylesheets/default.css.sass

```diff
@@ -34,9 +34,11 @@
 
 @import open_project_global/all
 
-@import foundation
+@import content/typography
```

Comment:
@ulferts the content partial should be loaded after the framework. If we need to make the variables available to `_settings.scss`, then move them into our own `open_project_global/_variables.sass`.